ang/civiawards/award-creation/directives/award.directive.js

```diff
@@ -16,6 +16,15 @@
   module.controller('CiviAwardCreateEditAwardController', function (
     $location, $q, $scope, $window, CaseTypeCategory, crmApi, crmStatus, getSelect2Value) {
     var ts = CRM.ts('civicase');
+    var DEFAULT_COMMUNICATION_ACTIVITY_TYPES = [
```

Comment:
Is it worth it having two separate arrays? how about having a single one called `DEFAULT_ACTIVITY_TYPES`?